homepage-trends/06.json

```diff
@@ -1,6 +1,7 @@
 {
   "chartConfig": {
-    "legendTitle": "Women Candidates",    
+    "legendTitle": "Women Candidates", 
+    "domain": [0, 40],   
```

Comment:
@vgeorge only review needed here is whether I can set the domain on this trend and have it render correctly.